test-suite/generated-src/inFileList.txt

```diff
@@ -20,6 +20,7 @@ djinni/varnames.djinni
 djinni/relative_paths.djinni
 djinni/array.djinni
 djinni/nscopying.djinni
+djinni/static_method_language.djinni
```

Comment:
Any purpose of having the infileList and the outfileList? I'm guessing it was copied from the original project, should we just delete them and stop setting the flags in the shell script?